x-pack/test/functional/services/transform_ui/wizard.ts

```diff
@@ -287,6 +294,10 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       );
     },
 
+    async enabledAdvancedPivotEditor() {
+      await testSubjects.click('transformAdvancedPivotEditorSwitch');
```

Comment:
Would be good to check that the click did something, e.g. call a new `assertAdvancedPivotEditorExists` or just
```
await testSubjects.existOrFail(`transformAdvancedPivotEditor`);
```

Reply:
Also, it should only click the switch if it's not already enabled.

Reply:
Fixed in 846aa25.